__tests__/app.test.js

```diff
@@ -61,43 +61,186 @@ describe('GET /api', () => {
       });
     })
 })
-// describe('GET /api/articles/:article_id', () => {
+describe('GET /api/articles/:article_id', () => {
+  describe('Basic request checks', () => {
+    test('returns status 200 on successful request', () => {
+      return request(app).get('/api/articles/1').expect(200);
+    });
+    test('response contains an object of 1 article with correct properties', () => {
+      return request(app)
+      .get('/api/articles/1')
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          title: 'Living in the shadow of a great man',
+          topic: 'mitch',
+          author: 'butter_bridge',
+          body: 'I find this existence challenging',
+          created_at: '2020-07-09T20:11:00.000Z',
+          votes: 100,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        })
+      });
```

Comment:
Instead of using this very brittle assertions which means if any new properties were added to article we can use the more flexible toMatchObject (https://jestjs.io/docs/expect#tomatchobjectobject) This matcher will check the object contains at least the properties in the object passed to this matcher. See the examples in the docs for a better visualisation